widgets/creativecommons-widget.php

```diff
@@ -80,8 +80,18 @@ public function print_license_footer() {
 		public function widget( $args, $instance ) {
 			$title = apply_filters( 'widget_title', $instance['title'] );
 			echo $args['before_widget'];
-			echo $args['before_title'] . $title . $args['after_title'];
+
+			/*
+			* To prevent rendering an empty heading tag (and related margin)
+			* only render title when it exists (and isn't an empty string)
+			*/
+			$title_exists = $title && $title != "";
+			
+			if ($title_exists)
+				echo $args['before_title'] . $title . $args['after_title'];
```

Comment:
I am always a fan of having curly brackets here as well (for the if).

Reply:
@TimidRobot just curious: can't we set this in the editorconfig?

Reply:
Good call. I'm having a bit of difficulty getting the VS Code phpcs extension to work. I think it is working now that I have connected to the WordPress Docker container with a proper PHP environment.

Reply:
I added curly braces to match the WordPress PHP code standards.